cmd/bad-key-revoker/main_test.go

```diff
@@ -61,11 +61,17 @@ func TestSelectUncheckedRows(t *testing.T) {
 
 	hashA, hashB, hashC := randHash(t), randHash(t), randHash(t)
 	insertBlockedRow(t, dbMap, fc, hashA, 1, true)
+	count, err := bkr.countUncheckedKeys()
+	test.AssertNotError(t, err, "countUncheckedKeys failed")
+	test.AssertEquals(t, count, 0)
 	_, err = bkr.selectUncheckedKey()
 	test.AssertError(t, err, "selectUncheckedKey didn't fail with no rows to process")
 	test.Assert(t, db.IsNoRows(err), "returned error is not sql.ErrNoRows")
 	insertBlockedRow(t, dbMap, fc, hashB, 1, false)
 	insertBlockedRow(t, dbMap, fc, hashC, 1, false)
+	count, err = bkr.countUncheckedKeys()
+	test.AssertNotError(t, err, "countUncheckedKeys failed")
+	test.AssertEquals(t, count, 2)
```

Comment:
optionally add a `test.AssertMetricWithLabelsEquals()` call here too

Reply:
Oh, nice, I didn't think to add this. I went ahead and added a test of this in the `invoke()` test cases since the metric is emitted as part of that loop.